config/.env.test

```diff
@@ -15,4 +15,6 @@ STRIPE_API_KEY=this_is_not_the_key
 STRIPE_SIGNING_SECRET=this_is_not_the_secret
 SES_REGION=this_is_not_the_secret
 SES_KEY=this_is_not_the_secret
-SES_SECRET=this_is_not_the_secret
\ No newline at end of file
+SES_SECRET=this_is_not_the_secret
+GUPSHUP_PARTNER_EMAIL=this_is_not_the_secret
```

Comment:
Let's move them to the Saas table since it belongs to a saas provider only.

Reply:
Yeah, I was thinking of the same but the table structure of the SaaS table is not that flexible. Also, I discussed the same with @dlobo

Reply:
You can create a column with jsonb column and put credentials there.